packages/utilities/psammead-assets/src/svgs.stories.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node, number, shape } from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { number as numberKnob, withKnobs } from '@storybook/addon-knobs';
+import notes from '../README.md';
+import { news, igbo, yoruba, thai, pidgin } from './svgs';
+
+const Svg = styled.svg`
+  display: block;
+  margin-top: 1.25rem;
```

Comment:
Can we not have `margin-top` here, or `padding` on the container please, it makes it clear the group doesnt have any top padding as it hugs the top of the page